python/tank/util/shotgun.py

```diff
@@ -763,7 +763,13 @@ def _group_by_storage(tk, list_of_paths):
     for path in list_of_paths:
 
         # use abstracted path if path is part of a sequence
-        abstract_path = _translate_abstract_fields(tk, path)
+        try:
+            abstract_path = _translate_abstract_fields(tk, path)
+        except TankError:
+            # This will happen if the path isn't valid due to a lack of
```

Comment:
for support and troubleshooting purposes, i think it would be good to add some debug logging here